File: src/gestorAplicacion/personajes/Guardian.java
/*
Implementación de la clase abstracta. define el método apoyar

Autor: Yeferson
*/
package personajes;

import complementario.Habilidad;

public abstract class Guardian {
	
	protected final String nombre;
	protected Habilidad habilidadEspecial;
	
	//Constructor para las clases hijas
	protected Guardian(String nombre, Habilidad habilidadEspecial) {
		this.nombre = nombre;
		this.habilidadEspecial = habilidadEspecial;
	}
	
	public abstract String apoyar(Peleador personaje);

	public String getNombre() {
		return nombre;
	}
	
	
}

File: Python/src/gestorAplicacion/personajes/guardian.py
'''
Implementación de la clase abstracta. define el método apoyar

Autor: Yeferson, Camilo, Edwin
'''

class Guardian:


    #Constructor para las clases hijas
    def __init__(self, nombre, habilidadEspecial):
        self.nombre = nombre
        self.habilidadEspecial = habilidadEspecial

    def apoyar(self, personaje):
        pass

    def getNombre(self):
        return self.nombre



